ci/install_openebs.sh

```diff
@@ -107,3 +107,21 @@ waitForDeployment ${JIVAREP} default
 kubectl get deploy -n openebs -l openebs.io/target=cstor-target
 CSTORTARGET=$(kubectl get deploy -n openebs -l openebs.io/target=cstor-target --no-headers | awk {'print $1'})
 waitForDeployment ${CSTORTARGET} openebs
+
+echo "---------------Testing deployment in pvc namespace---------------"
+kubectl create -f ./ci/volume/cstor/service-account.yaml
+
+kubectl create -f ./ci/volume/cstor/sc.yaml
+
+echo "---------------Creating in pvc---------------"
+kubectl create -f ./ci/volume/cstor/pvc.yaml
```

Comment:
Move this create stuff to `setup_env.sh` file, or add the example files to openebs/openebs repo and then use them here.